### PR TITLE
Update to FML locator rewrite

### PR DESCRIPTION
**Pull request by shartte**
See https://github.com/neoforged/FancyModLoader/pull/112

### CODE REVIEW

File: src/main/resources/assets/neoforge/lang/en_us.json [hunk 1]
```diff
@@ -90,6 +91,12 @@
   "fml.modloading.brokenfile.invalidzip": "File {2} is not a jar file",
```
Comment (Technici4n) [inline src/main/resources/assets/neoforge/lang/en_us.json:94]: You should add `fml.modloading.brokenfile.minecraft_forge` as well.

File: src/main/resources/assets/neoforge/lang/en_us.json [hunk 2]
```diff
@@ -85,7 +85,7 @@
   "fml.modloading.cycle": "Detected a mod dependency cycle: {0}",
   "fml.modloading.failedtoprocesswork":"{0,modinfo,name} ({0,modinfo,id}) encountered an error processing deferred work\n\u00a77{2,exc,msg}",
   "fml.modloading.brokenfile": "File {2} is not a valid mod file",
-  "fml.modloading.brokenfile.oldforge": "File {2} is for an older version of NeoForge and cannot be loaded",
+  "fml.modloading.brokenfile.minecraft_forge": "File {2} is for Minecraft Forge, or an older version of NeoForge and cannot be loaded",
```
Comment (Technici4n) [inline src/main/resources/assets/neoforge/lang/en_us.json]: you removed `oldforge` but it still exists in FML?

Comment (shartte) [reply src/main/resources/assets/neoforge/lang/en_us.json]: Oh right for mcmod.info, nevermind.

Comment (shartte) [reply src/main/resources/assets/neoforge/lang/en_us.json]: Fixed

**Review by Matyrobbrt — APPROVED** [review]

**Pull request merged**